cura_app.py

```diff
@@ -15,6 +15,10 @@
 import argparse
 import faulthandler
 import os
+
+# set the environment variable QT_QUICK_FLICKABLE_WHEEL_DECELERATION to 5000 as mentioned in qt6.6 update log to overcome scroll related issues
+os.environ["QT_QUICK_FLICKABLE_WHEEL_DECELERATION"] = str(int(os.environ.get("QT_QUICK_FLICKABLE_WHEEL_DECELERATION", "5000")))
```

Comment:
Won't this produce a value error when `QT_QUICK_FLICKABLE_WHEEL_DECELERATION` is set to a non number string.